docs-ref-conceptual/dotnet-howto-migrate-app-service.md

```diff
@@ -1,54 +1,108 @@
 ---
-title: Migrate an ASP.NET web application to Azure App Service
-description: Learn how to migrate an ASP.NET web application from on-premises to Azure App Service.
-keywords: Azure .NET, ASP.NET, App Service, Web App, migrate, migration
+title: Migrate your .NET web app or service to Azure App Service
+description: Learn how to migrate a .NET web app or service from on-premises to Azure App Service.
+keywords: Azure .NET, ASP.NET, WCF, App Service, Web App, migrate, migration
 author: camsoper
 manager: wpickett
 ms.author: casoper
-ms.date: 11/15/2017
+ms.date: 07/16/2018
 ms.topic: article
 ms.technology: azure
 ms.devlang: dotnet
 ms.service: app-service
 ms.custom: devcenter
 ---
 
-# Migrate an ASP.NET web application to Azure App Service
+# Migrate your .NET web app or service to Azure App Service 
 
-[App Service](https://docs.microsoft.com/azure/app-service/app-service-web-overview#why-use-web-apps) is a fully-managed compute platform service that is optimized for hosting scalable websites and web applications. This document provides information on how to lift-and-shift an existing application to Azure App Service, modifications to consider, and additional resources for moving to the cloud.
+Most ASP.NET websites (Webforms, MVC) and services (Web API, WCF) can move directly to Azure App Service with no changes. Some may need minor changes while others may need some refactoring.
 
-Ready to get started? [Publish your ASP.NET + SQL application to Azure App Service](https://go.microsoft.com/fwlink/?linkid=863214).
+## Port Bindings
 
-# Preparation   
-* [How to determine if your app qualifies for App Service](https://azure.microsoft.com/downloads/migration-assistant/)
-* [Moving your database to the cloud](https://go.microsoft.com/fwlink/?linkid=863217)
+Azure App Service supports port 80 for HTTP and port 443 for HTTPS traffic.
+
+For WCF, the following bindings are supported:
+
+Binding | Notes
+--------|--------
+BasicHttp | 
+WSHttp | 
+WSDualHttpBinding | Requires message security. Web socket support must be enabled for the App Service.
```

Comment:
Message security is a requirement of the binding, not of App Service, so may not be worth calling out in this table. I called it out in my notes because message security looks like it should probably work on App Service but was so difficult to validate that I gave up.

(WebSocket support is worth calling out here because `WSDualHttpBinding` only supports duplex contracts.)